templates/messages/2023-08-01-tu-b-av.html

```diff
@@ -70,7 +70,7 @@
           Show your love for Sefaria this Tu B’Av!
         </span>
         <span class="int-he">
-          !הביעו את אהבתכם לספריא לרגל ט"ו באב
+הביעו את אהבתכם לספריא לרגל ט"ו באב!
```

Comment:
Is the punctuation in the right location here?

Reply:
Shockingly it is. 

Here's the same thing if you convert the string to JS numeric strings:

`\u{5D4}\u{5D1}\u{5D9}\u{5E2}\u{5D5} \u{5D0}\u{5EA} \u{5D0}\u{5D4}\u{5D1}\u{5EA}\u{5DB}\u{5DD} \u{5DC}\u{5E1}\u{5E4}\u{5E8}\u{5D9}\u{5D0} \u{5DC}\u{5E8}\u{5D2}\u{5DC} \u{5D8}"\u{5D5} \u{5D1}\u{5D0}\u{5D1}!`

One of the most challenging things about dealing w/ rtl content is that often punctuation appears to be incorrectly placed when it is, or incorrectly placed when it's not.

Reply:
Yeah, I don't think GitHub has properly implemented dealing with Hebrew... lol